deploy-board/deploy_board/webapp/cluster_view.py

```diff
@@ -76,9 +76,10 @@ def post(self, request, name, stage):
             cluster_info = json.loads(request.body)
 
             log.info("Create Capacity in the provider")
-            if 'spiffe_id' in cluster_info:
-                log.error("Teletraan does not support user to change spiffe_id %s"  % cluster_info['spiffe_id'])
-                raise TeletraanException("Teletraan does not support user to create spiffe_id")
+            if 'configs' in cluster_info:
```

Comment:
looks like we change the behavior here, was it a bug before that spiffe_id supposed to be in cluster_info['config'] but not in cluster_info?

Reply:
Yes, it was a bug.